src/Microsoft.ML.Core/Data/ModelHeader.cs

```diff
@@ -351,10 +351,19 @@ public static void CheckVersionInfo(ref ModelHeader header, VersionInfo ver)
             Contracts.CheckDecode(header.ModelSignature == ver.ModelSignature, "Unknown file type");
             Contracts.CheckDecode(header.ModelVerReadable <= header.ModelVerWritten, "Corrupt file header");
             if (header.ModelVerReadable > ver.VerWrittenCur)
-                throw Contracts.ExceptDecode("Cause: ML.NET {0} cannont read component '{1}' of the model, because the model is too new.\n" +
+                throw Contracts.ExceptDecode("Cause: ML.NET {0} cannot read component '{1}' of the model, because the model is too new.\n" +
                                 "Suggestion: Make sure the model is trained with ML.NET {0} or older.\n" +
```

Comment:
I would reverse the suggestion as otherwise we are implying the user should always use older version of ML․NET.
```suggestion
                                "Suggestion: Update the model execution version to ML.NET {1} or newer.\n" +
```